README.md

```diff
@@ -17,6 +17,16 @@ fun main(args: Array<String>) {
 }
 ```
 
+If you need, you can use HTTP or SOCKS proxy
+
+```kotlin
+fun main(args: Array<String>) {
+    val bot = bot {
+        proxy = Proxy(Proxy.Type.SOCKS, InetSocketAddress("hostname", port))
+    }
+}
+```
+
```

Comment:
I would move this off the main README. The idea is to keep the main README as simple as possible and to add more detailed documentation within the docs directory in other MD files. What do you think about this?